pages/while-job-hunting.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Job Hunting
+layout: default
+---
+
+<script src="https://gist.github.com/rossghill/3aceb0b18c2efa097538594ef277df21.js"></script>
```

Comment:
We should probably add a bit to credit Ross for this - wdyt?

Reply:
Yeah i thought so too, thought the script would add the author